### ISSUE TITLE
parse_claymore_status.awk does not process dcoin correctly

### ISSUE THREAD

**Issue by wishbone1138** [rodneymo/rig-monitor#14]
parse_claymore_status.awk does not process dcoin correctly (SC in my case):
```
#READ DCR/SC/LBC/PASC GPU HASHRATE E.G.
#  SC: GPU0 43.678 Mh/s 
/^  (DCR|SC|LBC|PASC):/ {
        #print $5,$9,$13,$17,$21,$25
	gpu_field=6
	while ( gpu_field < NF ) {
		_index = gpu_field/6 - 1
		gpu[_index,"HASHRATE_DCOIN"]=$gpu_field
		gpu_field+=4	
	}
}
```
I think this one is a bug, you increment by 4, but divide by 6 every time.  I used a _index variable and just ++ it.

```
 READ DCR/SC/LBC/PASC TOTAL HASHRATE and SHARE INFO  E.G.
  SC - Total Speed: 43.678 Mh/s, Total Shares: 0, Rejected: 0 
/^  (DCR|SC|LBC|PASC) - Total Speed: / { 
	#print $8,$14,$18
	current_hashrate_dcoin = $8
	total_shares_dcoin = $14
	rejected_shares_dcoin = $18
}
```

Mine actually looks like your ETH example prior to this block:
 SC - Total Speed: 43.678 Mh/s, Total Shares: 0(#+#+#+#+etc), Rejected: 0 
I just used the same code from ETH to fix this one (with dcoin variables)

**Comment by rodneymo** [comment]
Thanks. I have split your problems into 2 different issues.

**Comment by wishbone1138** [comment]
Sounds good. I pushed some possible fixes to you for two of them.  I haven't had time yet to clean up the third.

**Comment by rodneymo** [comment]
@wishbone1138 I pushed your fixes to the development branch.
I have also added support to count the shares/GPU, but I have not been able to test it. I run my rigs in single mode and the PC I use to test only has 1 card.
Could you attach claymore's output mining ETH+SC? Preferably with rejected shares so I can validate it.

**Comment by wishbone1138** [comment]
You have a typo my friend.  I'll push a patch to your development branch (hopefully I do that right).  Took me forever to find it.

Unfortunately I don't have any rejected shares, so I can't test that part.  I don't think I've never seen one in claymore on my setup.

**Comment by rodneymo** [comment]
Fixed in 1.2.0
